cle/backends/backend.py

```diff
@@ -180,9 +180,6 @@ def __init__(
                 continue
             log.critical("Deprecation warning: the %s parameter has been renamed to %s", k, k[7:])
 
-        if kwargs != {}:
-            log.warning("Unused kwargs for loading binary %s: %s", self.binary, ", ".join(kwargs.keys()))
-
```

Comment:
Why was this removed?

Reply:
Because they are not actually unused; a backend can use saved arguments from from `load_args` any time it wants.